src/Cawbird.vala

```diff
@@ -94,13 +94,23 @@ public class Cawbird : Adw.Application {
    * Initialize the client and open the first window.
    */
   protected override void activate () {
+    Backend.Client client;
     // Initializes the backend client
-    var client = new Backend.Client (Config.PROJECT_NAME,
-                                     "https://github.com/CodedOre/NewCaw",
-                                     "cawbird://authenticate");
+    try {
+      client = new Backend.Client("id",
+        Config.PROJECT_NAME,
+        "https://github.com/CodedOre/NewCaw",
+        "cawbird://authenticate");
+    } catch (Error e) {
+      critical (@"Failed to load program state: $(e.message)");
+      return;
+    }
```

Comment:
Is the try-catch still needed?

Reply:
Possibly not. But the compiler didn't tell me that it wasn't needed, so I left it there in case there was an exception that I was missing. I'll remove it and see how well it works.